buckw

```diff
@@ -66,7 +66,7 @@ OKBUCK_DIR="$SCRIPT_DIR/.okbuck"
 MAX_DISPLAY_CHANGES=10
 WATCHMAN_TIMEOUT=10
 WATCHMAN_FAILED="WATCHMAN FAILED"
-BUCK_BINARY_DIR="$OKBUCK_DIR/cache/buck_binary"
```

Comment:
Lets leave it `$OKBUCK_DIR`. This allows buckw to be called from any location outside root.